examples/flutter_gallery/lib/demo/text_field_demo.dart

```diff
@@ -79,26 +112,19 @@ class TextFieldDemoState extends State<TextFieldDemo> {
             new Input(
               hintText: 'What do people call you?',
               labelText: 'Name',
-              formField: new FormField<String>(
-                // TODO(mpcomplete): replace with person#name=
-                setter:  (String val) { person.name = val; },
-                validator: _validateName
-              )
```

Comment:
My motivation with the FormField stuff was to remove the need for the app developer to manage any sort of intermediate framework objects for form data. They would manage their app-specific data struction (e.g. Person) and the framework holds the InputValues and knows how to update the app-specific structure automatically.

My original vision was something that let me do:
  new Input(
    labelText: 'Name',
    mapsTo: person#name
  )
then person.name is the Real Truth for what is shown in the Input, and is updated whenever the Input changes. Dart doesn't quite let us do that, so supplying a `setter` was the next-best thing. There's a proposal for adding tear-offs to Dart, but AFAIK it isn't official adpoted. If it were, we could do:
  new Input(
    labelText: 'Name',
    setter: person#name=
  )

which seems close enough. IMO that's the direction we should be heading.

Maybe we should have a sub-class of Input that allows you to do the short-hand with conversion between InputValue and String, rather than a separate FormField argument to Input?